src/main/java/baron/Baron.java

```diff
@@ -0,0 +1,23 @@
+package baron;
+
+import baron.Managers.TaskManager;
+
+import java.util.Scanner;
+
+public class Baron {
+    public static void main(String[] args) {
```

Comment:
I feel that there could be a Javadoc comment to mention how this method would be the main entrypoint of your application?